wtpython/display.py

```diff
@@ -199,19 +201,7 @@ async def on_startup(self, event: events.Startup) -> None:
         self.sidebar: Sidebar = Sidebar("sidebar", SO_RESULTS)
         self.body: ScrollView = ScrollView(self.create_body_text())
 
-        footer.add_key("q", "Quit")
-        footer.add_key("←", "Prev Q")
-        footer.add_key("→", "Next Q")
-        footer.add_key("s", "Show Questions")
-        footer.add_key("t", "Toggle Traceback")
-        footer.add_key("d", "Open Browser")
-        footer.add_key("f", "Google")
-        footer.add_key("i", "Report Issue")
-
-        await self.set_focus(self.body.page)
```

Comment:
Is this done on purpose? Does textual not automatically focus the ScrollView when the app is launched?